Mixpanel/Mixpanel.m

```diff
@@ -228,6 +234,9 @@ - (void)identify:(NSString *)distinctId
         }
         [self archiveProperties];
     });
+#if defined(MIXPANEL_WATCH_EXTENSION)
+    [self flush];
```

Comment:
since the watch doesn't flush on a timer identify wouldn't be sent until the next track event. maybe that's okay though? if that's okay then we don't need this